package.json

```diff
@@ -8,8 +8,8 @@
     "bootstrap": "run-s bootstrap:sponsors build",
     "bootstrap:sponsors": "node ./scripts/download-sponsors.js",
     "build": "run-s build:codemirror6 build:repl build:docusaurus build:redirects",
-    "build:docusaurus": "cd website && yarn && yarn build && mv build babel && mkdir build && mv babel build/babel",
-    "build:redirects": "cp ./_redirects ./website/build/babel",
```

Comment:
`cp` does not exist on Windows, but we can use node's [fs.cpSync() function](https://nodejs.org/api/fs.html#fscpsyncsrc-dest-options), which is available since v16.7.0.

This is how removing and copying files are done in the SvelteKit repo too: [link to source](https://github.com/sveltejs/kit/blob/main/packages/adapter-netlify/package.json#L36)